pkg/controller/node/nodecontroller.go

```diff
@@ -189,6 +191,15 @@ func NewNodeController(
 			UpdateFunc: func(_, obj interface{}) { nc.maybeDeleteTerminatingPod(obj) },
 		},
 	)
+
+	nodeEventHandlerFuncs := framework.ResourceEventHandlerFuncs{}
+	if nc.allocateNodeCIDRs {
+		nodeEventHandlerFuncs = framework.ResourceEventHandlerFuncs{
```

Comment:
if the nodeController restarts, ResourceEventHandlers would observe some Add Event. So we need mark the CIDR has been occupied if `node.Spec.PodCIDR != ""`, sorry for the missing.